2014-06-23 15:49:20,319 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:49:37,062 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:49:41,903 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:49:54,878 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:50:33,775 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:52:22,667 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:54:34,532 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:54:36,947 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:54:39,286 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 15:54:48,605 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
2014-06-23 16:15:18,950 ERROR exception
Traceback (most recent call last):
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\scapp\views\reimbursement\fysp.py", line 108, in fysp_check
    if request.form['fail_reason']:
  File "D:\Development\eclipse-jee_workspace\OA_NEW_Git\virtualenv32\lib\site-packages\werkzeug\datastructures.py", line 391, in __getitem__
    raise exceptions.BadRequestKeyError(key)
BadRequestKeyError: 400: Bad Request
